src/mbdynprecice/_tools.py

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
```

Comment:
Are these post-processing tools meant for the user? Are they developed specifically for this case, or is this something coming from MBDyn? Some documentation would be very useful here.